ncpi_model_forge/ingest_plugin/ncpi_forge_fhir.py

```diff
@@ -0,0 +1,19 @@
+from requests import RequestException
+from ncpi_fhir_utility.client import FhirApiClient
+from ncpi_model_forge.ingest_plugin.ncpi_patient import Patient
+
+all_targets = [
+  Patient
+]
+
+
+# Basic example submit method
+def submit(host, entity_class, body):
+    client = FhirApiClient(base_url=host, auth=("admin", "password"))
+    success, result = client.send_request(
+        "POST", f"{host}/Patient", json=body
+    )
```

Comment:
Unless we use `"PUT"`, the FHIR server will assign a random ID to a resource. So, we may want to do this:

```python
verb = "POST"
url = f"{host}/Patient"

if body.get("id"):
    verb = "PUT"
    url = f"{url}/{body.get('id')}"

success, result = client.send_request(verb, url, json=body)
```

Reply:
For the purposes of the tutorial, I don't think we care.

Reply:
It may be a FHIR thing and PUT on FHIR is `CREATE IF NOT EXIST OR UPDATE` (https://www.hl7.org/fhir/http.html#update). So, we may not have to bring such complexity.